README.md

```diff
@@ -109,7 +132,7 @@ Write this grammar using EBNF with common extensions
 
 ## A2.3
 
-«replace this with your answer»
+See StateMachineDiagram.png
```

Comment:
# 5

It is correct, but it has an unnecessary state: you could remove S2 and have S1 loop back to itself on an adjective.